src/StreamJsonRpc.Shared/JsonRpc.cs

```diff
@@ -654,6 +661,38 @@ protected virtual Task<ReturnType> InvokeCoreAsync<ReturnType>(int? id, string t
                 }
             }
         }
+        
+        private static Dictionary<string, List<string>> GetMethodToAttributeMap(object target)
```

Comment:
Can you add xml doc comments to this method, including a thorough description of what the returned dictionary contains?